### ISSUE TITLE
Improve data model

### ISSUE THREAD

**Issue by plhnk** [liferay-design/liferay.design#641]
We want to reuse the data generated from the gdocs — currently it looks like this:

```json
"currentCategory": {
    "department": "$department",
    "displayName": "$currentCategory",
    "milestones": [
        {
            "summary": "$summary",
            "signals": [
                "$signals"
            ]
        }
    ],
    "description": "$description"
}
```

This is problematic because there is a lot of redundancy, and it makes querying unnecessarily complex.

The desired model should be something like this:

```json
"vertical": "$department",
"skills": [
    {
        "label": "$label",  //currentCategory
        "description": "$description", 
        "levels": [
            {
                "summary": "$levelSummary",
                "signals": [
                    "$signals"
                ]
            }
        ]
    }
]
```

This is found in the `generateTracks.js` file — but also the changes here will impact the whole app, requiring all of the queries to be rewritten per the new model.

**Comment by plhnk** [comment]
note this has been moved to `src/utils/generateTracks`

**Comment by plhnk** [comment]
thanks to @jwu910 this has been resolved in #644

**Comment by jwu910** [comment]
Always fun seeing new projects. Let me know if theres anything else you need!
